app/custom_ante/wasm.go

```diff
@@ -0,0 +1,26 @@
+package custom_ante
+
+import (
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// TODO: Replace with the correct height
+const DenyMsgStoreCodeAfter = 1000000
+
+func DenyMsgStoreCode(next sdk.AnteHandler) sdk.AnteHandler {
+	return func(ctx sdk.Context, tx sdk.Tx, simulate bool) (_ sdk.Context, _ error) {
+		if ctx.BlockHeight() > DenyMsgStoreCodeAfter {
+			for _, msg := range tx.GetMsgs() {
+				if _, ok := msg.(*wasmtypes.MsgStoreCode); ok {
```

Comment:
security issue: authz nested messages could bypass this

Also, why? modify CW params to be permissioned instead of this